test_files/multiThreadTest1.h

```diff
@@ -0,0 +1,3 @@
+void func1()
```

Comment:
No. I mean `void func1(void);`. DStep currently doesn't handle the body of a function. It don't want there to be any confusion.